test/controllers/file_submissions_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class FileSubmissionsControllerTest < ActionController::TestCase
+
+  def setup
+    create(:active_game)
+  end
+
+  test 'guest and user cannot access file submissions' do
+    user = create(:user)
+    challenge = create(:file_submission_challenge)
+    file_submission = create(:file_submission,
+                              challenge_id: challenge.id,
+                              user_id: user.id,
+                              submitted_bundle: File.open(Rails.root.join('test/files/test_submission.zip')),
+                              description: "Text",
+                              demoed: false
+                            )
+    assert_raises(ActiveRecord::RecordNotFound) do
+      get :submitted_bundle, params: { id: user.id } # Nobody is signed in
```

Comment:
The ID here should not be the user's ID, it should be the bundle ID based on code in `def submitted_bundle`.